### DISCUSSION TITLE
Is raspAP compatible with nm-connection-editor ?

### DISCUSSION THREAD

**Discussion by mh4ckt3mh4ckt1c4s** [RaspAP/raspap-webgui#1098] [Q&A] [answered]
Hello everyone, 

I've just started to play with Raspberry (I have a raspberry 4) and I wanted to create an AP with it. However, the wired connection I use to access Internet need authentication (802.1X PEAP). I looked around and found out there is nothing yet on raspian to deal with such authentication, so I installed `nm-connection-editor` via `apt` and used it to authenticate. 

Problem is, it seems to cause a lot of issues with raspAP (with the default config, or some custom I tried). I don't know exactly why, but I strongly suspect the components of raspAP to interfere with my `nm-connection-editor` config. Bottom line is the AP is really buggy, disappearing from time to time, and making me lose the connection when connected to the AP or even directly on my Raspberry. In the network panel in raspAP, the eth0 interface is not even listed in the internet connection. Also, my computer is giving me a strange IP address of 169.254.x.x in addition of my public IP address (I never had this kind of IP before).

So I was wondering if someone out there knew how was exactly working `nm-connection-editor` and if it was possible that it is interfering with the various components of raspAP, and if there is a possibility to fix this (if I'm not mistaken in the problem).

Thanks for the answers you can bring me and for this project.

**Comment by billz** [comment, marked as answer]
Short answer: no, nm-connection-editor is not compatible with RaspAP.

More precisely, you are attempting to use two different network managers, which are in conflict. RPi OS uses dhcpcd as its network configuration solution. Your package uses NetworkManager and `/etc/network/interfaces`. 

This is not a RaspAP problem, it's an issue fundamental to Linux networking.

**Reply by mh4ckt3mh4ckt1c4s** [reply]
Hi, thanks for the answer. I looked around for a possiblity to solve the problem by using only dhcpcd as my network config solution but I was unable to found any resource dealing with the dhcpcd config of an 802.1X authentication. Is that possible at all with dhcpcd, or is RaspAP just not usable with Internet authentication ?
